src/Model/Translatable/TranslatableMethodsTrait.php

```diff
@@ -117,6 +128,9 @@ public function getDefaultLocale(): string
         return $this->defaultLocale;
     }
 
+    /**
+     * @return class-string<T>
+     */
     public static function getTranslationEntityClass(): string
     {
         return static::class . 'Translation';
```

Comment:
PHPStan is complaining here as the class is ensured to exist.
I see two alternatives (other than ignoring the errors) :
- Remove the `@return class-string<T>`.
- Add a check on the class name to check it exists (and throw an exception if it doesn't exist).

Reply:
The `class-string<>` brings value here for static analysis as it can detect when class name is invalid / not existing.

I would be in favor to keep it, but it will require to add boilerplate on the default implementation code to check that the class exists before and maybe also check that's it's an implementation of `TranslationInterface`. When overriding the trait method, simply returning `class::name` will allow static analyzer to do its job and ensure the class is valid and an instance of TranslationInterface. WDYT?

Reply:
To be honest... I don't know. The reason is I don't have time to maintain this package and give the propper care it needs.

It step up few years ago, when we used it at work and needed new PHP version support. Yet haven't used it since.

I have other OS project I want to focus on.

Saying that, I'll be stepping down as maintainer and give space someone to take over.